mcodingbot/plugins/python.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+import crescent
+import hikari
+
+from mcodingbot.utils import Plugin
+
+plugin = Plugin()
+
+pep_regex = re.compile(r"pep *\d{1,4}", re.IGNORECASE)
```

Comment:
whitespace should be optional

Reply:
It is optional. That's what the `*` is for.

Reply:
oh my bad, didn't see it